packages/core/content-manager/admin/src/exports.ts

```diff
@@ -6,7 +6,10 @@
 
 export { buildValidParams } from './utils/api';
 
-export { useDocument as unstable_useDocument } from './hooks/useDocument';
+export {
+  useDocument as unstable_useDocument,
+  useDoc as unstable_useDoc,
```

Comment:
there is a comment where useDoc is defined that indicate it's for internal use only. we should probably update the comment if the plan has changed

Reply:
I missed that comment. I'll do a mix of the 2 by export a wrapper as we definitely need to expose sth of that flavor regardless 👍

Reply:
nice, maybe this is a chance to find a better hook name then 😁

Reply:
any suggestion ? :D

Reply:
tough one! something like `useEditViewData`? It's pretty close to the v4 hook it replaces, and makes it clear that it only works on the edit view page

Reply:
the pbl is it's completly missing the data 🤔

Reply:
what is it returning then?

Reply:
I updated this to include usefull context for plugin developers. let me know what you think :)